resources/projectsyncup-design-document.md

```diff
@@ -0,0 +1,179 @@
+# Design Document
+
+## ProjectSyncUp Design
+
+## 1. Problem Statement
+
+_The objective of this project is to create a web application using Java, that helps streamline collaboration among team members working on projects together. The app will allow managers to create projects and assign tasks, while team members can track, prioritize and update progress of their assigned tasks, resulting in an efficient tool that increases project transparency for teams while working together._
+
+## 2. Project Scope
+
+### 3. Use Cases
+
+U1. As a manager, I want to create a new project with a given name.
+
+U2. As a manager, I want to create a new, empty task with a given task name, description, due date, and list of assigned team members.
+
+U3. As a manager, I want to add a task, to a project with a given name.
```

Comment:
I think you can/should combine U2 and U3. You're never going to create an "empty" task, it's always going to be part of a project.